cmd/build/main.go

```diff
@@ -36,10 +37,11 @@ func check(err error) {
 }
 
 type BuildConfig struct {
-	Name      string               `json:"name"`
-	OSTree    *ostree.ImageOptions `json:"ostree,omitempty"`
-	Blueprint *blueprint.Blueprint `json:"blueprint,omitempty"`
-	Depends   interface{}          `json:"depends,omitempty"` // ignored
+	Name         string                     `json:"name"`
+	OSTree       *ostree.ImageOptions       `json:"ostree,omitempty"`
+	Blueprint    *blueprint.Blueprint       `json:"blueprint,omitempty"`
+	Subscription *subscription.ImageOptions `json:"subscription,omitempty"`
```

Comment:
Wouldn't it make sense to add the the whole `distro.ImageOptions` instead?

Reply:
Probably a good idea yeah.  Will need to change the test configs for ostree.  But good idea.

Reply:
Done!